torch/types.py

```diff
@@ -57,7 +69,7 @@
 # bool or SymBool
 BoolLikeType: TypeAlias = Union[bool, SymBool]
 
-py_sym_types = (SymInt, SymFloat, SymBool)
+py_sym_types: tuple = (SymInt, SymFloat, SymBool)
```

Comment:
```suggestion
py_sym_types: tuple[type[SymInt], type[SymFloat], type[SymBool]] = (SymInt, SymFloat, SymBool)
```
Got to be a better way to infer this...